docs/source/python/extensions.rst

```diff
@@ -92,17 +92,17 @@ In addition to extension classes, there are a few helper functions to ease handl
 
    A dictionary mapping of a unique key to an extension class::
 
-      >>> KEY_TO_EXTENSION['authority_information_access']
-      <class 'django_ca.extensions.extensions.AuthorityInformationAccess'>
+      >>> KEY_TO_EXTENSION['authority_information_access'].__name__
+      'AuthorityInformationAccess'
 
 .. autoattribute:: django_ca.extensions.OID_TO_EXTENSION
    :annotation: {<ObjectIdentifier(oid=...): <class 'django_ca.extensions...>, ...}
 
    A dictionary mapping of OIDs to extension classes::
 
       >>> from cryptography.x509.oid import ExtensionOID
-      >>> OID_TO_EXTENSION[ExtensionOID.BASIC_CONSTRAINTS]
-      <class 'django_ca.extensions.extensions.BasicConstraints'>
+      >>> OID_TO_EXTENSION[ExtensionOID.BASIC_CONSTRAINTS].__name__
+      'BasicConstraints'
```

Comment:
Dito.

Reply:
Looking at this, my statement was incorrect: doctests are run for all Python versions.

However, I'm personally fine with running them only for the newest python version. Doctests are spretty primitive and only run to ensure that they are not "totally wrong". They are still most important as documentation. So i think it's better to run them only on the newest version, e.g.

```
def load_tests(loader, tests, ignore):
    if not settings.NEWEST_PYTHON:
        return

    docs_path = os.path.join(settings.DOC_DIR, 'python', 'extensions.rst')
    tests.addTests(doctest.DocTestSuite('django_ca.extensions'))
    tests.addTests(doctest.DocFileSuite(docs_path, module_relative=False))
```

Reply:
> Doctests run (for this and other reasons) only on the newest version on Python. So this should not be an issue. 

Ah, all right.

> I'm personally fine with running them only for the newest python version.

All right. I will add this to all the `load_test` functions. Thanks.

Reply:
Done.

Reply:
This kind of workaround introduces another issue, see https://github.com/mathiasertl/django-ca/pull/69#issuecomment-782363892.